Assets/Scripts/Player/Input Parser/InputParser.cs

```diff
@@ -27,13 +32,31 @@ private void OnDisable()
         private void AddListeners()
         {
             _inputActionAsset["MoveTowards"].performed += MoveTowards;
+            _inputActionAsset["Interact"].performed += Interact;
         }
 
         private void RemoveListeners()
         {
             _inputActionAsset["MoveTowards"].performed -= MoveTowards;
+            _inputActionAsset["Interact"].performed -= Interact;
         }
 
         private void MoveTowards(InputAction.CallbackContext context) => _playerMovement.StartMovingEvent();
+
+        private void Interact(InputAction.CallbackContext context)
+        {
+            Ray ray = _mainCamera.ScreenPointToRay(SetMousePos());
+            Physics.Raycast(ray.origin, ray.direction *10, out RaycastHit hit);
+            
+            if (hit.collider
+                && hit.collider.CompareTag(INTERACTABLE_TAG))
+                hit.collider.transform.GetComponent<InteractableObject>().onInteract.Invoke();
```

Comment:
Je doet hier een `GetComponent` dat begrijpelijk is. Maar als je 2 keer de zelfde klikt dan pakt ie elke keer opnieuw. Graag de opslaan welke je klik en dan als je de zelfde klik dat je dan geen `GetComponent` doet.